definitions/private-law/json/AuthorisationCaseState.json

```diff
@@ -397,5 +397,26 @@
     "CaseStateID": "GATE_KEEPING",
     "UserRole": "caseworker-caa",
     "CRUD": "CRU"
+  },
```

Comment:
We need to add access to caseType as well via authorisationCaseType.json

Reply:
done